app/people/emeritus-faculty/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { getMockEmeritusFacultyList } from '@/apis/faculty';
+
+import PageLayout from '@/components/layout/pageLayout/PageLayout';
+import PeopleRow from '@/components/people/PeopleRow';
+
+export default async function EmeritusFacultyPage() {
+  const { facultyList } = await getMockEmeritusFacultyList();
+  return (
+    <PageLayout title="역대 교수진" titleType="big">
```

Comment:
```suggestion
    <PageLayout title="역대 교수진" titleType="big" titleMargin="mb-16">
```
밑에 다른 페이지들에도 마진 바텀이 빠져있는 것 같은데 확인 부탁!